.github/workflows/release.yml

```diff
@@ -56,11 +56,25 @@ jobs:
         with:
           print_hash: true
 
+      - name: Parse version and output PyPI URL
+        id: version
+        # run littlepay first, so `littlepay -v` doesn't create the config file (which outputs a message)
+        run: |
+          littlepay
+          pip install pyproject.toml
```

Comment:
Wait so if the above line to just run `littlepay` works, does it need to be installed again?

Also I created #32 to help us avoid this in the future (not a priority for now though).

Reply:
Ah, good catch. It doesn't work - see this run I just did: [`littlepay: command not found`](https://github.com/cal-itp/littlepay/actions/runs/8267643338/job/22618563960#step:8:18)

I had been troubleshooting the issue in my own separate repo where the workflow set-up is a little different (it installs littlepay in a separate step).

I'll fix this, thank you!

Reply:
Was able to get a run where the URL is good: https://github.com/cal-itp/littlepay/actions/runs/8267783473/job/22619068892#step:9:17

Going to drop [the commit](https://github.com/cal-itp/littlepay/pull/31/commits/99090f6958d6358e6a56f9dc2d632d2c679d8fbc) that temporarily disabled tests/environment protections/publishing